unified_targets/configs/AIRF7.config

```diff
@@ -0,0 +1,88 @@
+
+board_name AIRF7
+manufacturer_id AIRB
```

Comment:
Why are you using the manufacturer ID for Airbot?

Reply:
As it is an Airbot board. See product link.